src/main/java/mate/academy/spring/model/Role.java

```diff
@@ -0,0 +1,68 @@
+package mate.academy.spring.model;
+
+import java.util.Objects;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "roles")
+public class Role {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Enumerated(value = EnumType.STRING)
+    @Column(unique = true)
```

Comment:
Add name column. Sth like "role" or "name", cause "roleName" as a column name is a bad idea